cli/script/command-parser.ts

```diff
@@ -212,6 +212,12 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
     .command("logout", "Log out of the current session", (yargs: yargs.Argv) => {
         isValidCommandCategory = true;
         isValidCommand = true;
+        yargs.usage(USAGE_PREFIX + " logout [--local <true|false>]")
```

Comment:
Ah I see, you were matching what was there before. I say yes for sure, the convention in CLI's (e.g. git) is definitely that you don't need to specify 'true' or 'false'.

If you don't mind, could you also update the README?